tests/test_colors.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+test_colors
+Test for colors 
+@author: iancharest
+"""
+
+import unittest
+from pyrsa.vis.colors import color_scale
+from pyrsa.vis.colors import rdm_colormap
+
+class ColorTests(unittest.TestCase):
```

Comment:
Perhaps you can add a quick test that actually tests the colors, with a dummy rdm for example, checking that cell 1 is brighter than cell2 or whatever the goal of the colormap is. Currently if the function just returns a bunch of nans, you'd never know

Reply:
working on it.